Base/TestURLSession.swift

```diff
@@ -213,28 +240,29 @@ private extension HttpMethod {
         case "PATCH":
             return .patch(nil)
         default:
-            BaseLog.testSupport.log(.error, "Invalid HttpMethod supplied: \(name)")
-            fatalError()
+            let message = "Invalid HttpMethod supplied: \(name)"
+            BaseLog.testSupport.log(.error, message)
+            fatalError(message)
         }
     }
 }
 
 extension URLMatch: EnvironmentRepresentable {
-    init(environmentRepresentation rep: [String:Any]) {
-        host = rep["host"] as! String
-        path = rep["path"] as! String
-        method = HttpMethod.from(methodName: rep["method"] as! String)
+    init(environmentRepresentation rep: [String: Any]) {
+        host = (rep["host"] as? String) ?? ""
```

Comment:
As this is test code I think it's more appropriate to fatal error instead of nil coalesce - also, some of this will likely crash if run without a value